diff --git a/node_modules/@vitejs/plugin-vue/dist/index.mjs b/node_modules/@vitejs/plugin-vue/dist/index.mjs
index 17b35e8..c093528 100644
--- a/node_modules/@vitejs/plugin-vue/dist/index.mjs
+++ b/node_modules/@vitejs/plugin-vue/dist/index.mjs
@@ -2262,6 +2262,9 @@ function deepEqual(obj1, obj2, excludeProps = []) {
   if (obj1 == null || obj2 == null || typeof obj1 !== "object") {
     return obj1 === obj2;
   }
+  if (obj1 === obj2) {
+    return true;
+  }
   const keys1 = Object.keys(obj1);
   const keys2 = Object.keys(obj2);
   if (keys1.length !== keys2.length) {
